CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Fixed
+- Adjusted the logic on the searchMetadata file to avoid URI malformed error on searches with % at the end and keeping the older logic working.
```

Comment:
```suggestion
- Adjusted the logic on the searchMetadata file to avoid URI malformed error on searches with % at the end and keep the older logic working.
```